res_syntax/tests/printer/expr/expected/UncurriedByDefault.res.txt

```diff
@@ -155,6 +155,3 @@ type callback6 = (ReactEvent.Mouse.t => unit) as 'callback
 let fooU = () => ()
 let fnU = (_x): (unit => unit) => fooC
 let aU = (() => "foo")->Ok
-
-let toIgnoredName = concatStrings("abc", _)
-Ok("_")->Belt.Result.map(() => concatStrings(_, "foo"))
```

Comment:
> Somehow roundtrip tests are not happy.

@cristianoc  Because the expected should be 
```
Ok("_")->Belt.Result.map(concatStrings(_, "foo"))
```

Reply:
Indeed that was the wrong fix.
Here https://github.com/rescript-lang/rescript-compiler/pull/6153